docs/develop/enable-nested-app-authentication-in-your-add-in.md

```diff
@@ -208,7 +208,14 @@ Nested app authentication enables SSO for applications that are nested inside of
 
 ## NAA supported accounts and hosts
 
-NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn’t support B2C scenarios. For preview, NAA is supported in Office on Windows and on Mac. For GA, NAA will also support Office on the web, and Outlook mobile on Android and on iOS.
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn’t support B2C scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
+
+| Application | Windows                                                               | Mac | Web          | iOS | Android |
```

Comment:
Consider indicating iPad vs iOS support (e.g., "iPad" or "Yes (iPad)" for WXP, "iOS" or "Yes (iOS)" for Outlook)

```suggestion
| Application | Windows                                                               | Mac | Web          | iOS/iPad | Android |
```